src/Squidex.Infrastructure/ComplexQuerySchemaValidationException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Squidex.Infrastructure
+{
+    public class ComplexQuerySchemaValidationException : Exception
```

Comment:
I have added this exception to the filter, so when the code throws this exception it will produce a 500 status code with an error message.
So the purpose was that I wanted to have something to say "you are returning the wrong schema in your plugin/query, fix it!"

Reply:
For a 500 you are fine, don't do anything. But when you want a specific error code / message you should add your exception handler.